CHANGELOG.md

```diff
@@ -2,33 +2,44 @@
 
 ## Current (in progress)
 
-- Dynamic API documentation, Enhancement to Pull #1542 - [#1542](https://github.com/opendatateam/udata/pull/1542)
-- Normalize resource.format (migration - :warning: need reindexing) [#1563](https://github.com/opendatateam/udata/pull/1563)
+### New features
+
 - Typed resources [#1398](https://github.com/opendatateam/udata/issues/1398)
-- [breaking] Enforce a domain whitelist when resource.filetype is file [#1567](https://github.com/opendatateam/udata/issues/1567)
 - Initial data preview implementation [#1581](https://github.com/opendatateam/udata/pull/1581) [#1632](https://github.com/opendatateam/udata/pull/1632)
-- Switch to PyPI.org for package links [#1583](https://github.com/opendatateam/udata/pull/1583)
 - Handle some alternate titles and alternate URLs on licenses for improved match on harvesting [#1592](https://github.com/opendatateam/udata/pull/1592)
-- Resource modals: use new single resource API call [#1547](https://github.com/opendatateam/udata/pull/1547)
-- Protocol-relative URLs support [#1599](https://github.com/opendatateam/udata/pull/1599)
-- Audience metrics: use only `views` [#1607](https://github.com/opendatateam/udata/pull/1607)
 - Allow to specify a dataset acronym [#1217](https://github.com/opendatateam/udata/pull/1217)
-- Add missing spatial granularities translations [#1636](https://github.com/opendatateam/udata/pull/1636)
-- Add resend confirmation email link to login screen [#1653](https://github.com/opendatateam/udata/pull/1653)
-- Refactoring of discussions code base [#1623](https://github.com/opendatateam/udata/pull/1623)
-- Fix Firefox custom error handling, part 2 [#1671](https://github.com/opendatateam/udata/pull/1671)
+- Starts using harvest backend `config` (validation, API exposition, `HarvestFilters`...) [#1716](https://github.com/opendatateam/udata/pull/1716)
 - The map widget can now be configured (tiles URL, initial position...) [#1672](https://github.com/opendatateam/udata/pull/1672)
+- New discussions layout [#1623](https://github.com/opendatateam/udata/pull/1623)
+- Dynamic API documentation, Enhancement to Pull #1542 - [#1542](https://github.com/opendatateam/udata/pull/1542)
+
+### Breaking changes
+
+- Normalize resource.format (migration - :warning: need reindexing) [#1563](https://github.com/opendatateam/udata/pull/1563)
+- Enforce a domain whitelist when resource.filetype is file [#1567](https://github.com/opendatateam/udata/issues/1567)
```

Comment:
See `RESOURCES_FILE_ALLOWED_DOMAINS` settings variable for details and configuration.